### PR TITLE
CAMS-283: e2e documentation

### PR DESCRIPTION
**Pull request by amorrow-flexion**
# Purpose

Provide developers documentation for running e2e tests locally

# Major Changes

- added e2e-tests.md outlining running end to end tests locally

# Testing/Validation

N/A

### CODE REVIEW

File: docs/operations/e2e-tests.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+In order to run playwright locally you will need to install playwright with the project dependencies. That can be done from your terminal within the test/e2e/ directory:
+
+`npx playwright install --with-deps`
+
+## Running E2E Tests
+
+### Setup for running against locally hosted application
+
+- When running playwright against the locally hosted application, you will wnt to ensure the NodeApi is running against a clean e2e database or the database needed for testing. `COSMOS_DATABASE_NAME=cams-e2e-<envHash>`
```
Comment (fmaddenflx) [inline docs/operations/e2e-tests.md:23]:
```suggestion
- When running playwright against the locally hosted application, you will want to ensure the NodeApi is running against a clean e2e database or the database needed for testing. `COSMOS_DATABASE_NAME=cams-e2e-<envHash>`
```

File: docs/operations/e2e-tests.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# End-2-End Tests
+
+## .env file
+
+To run E2E tests on the CAMS app from your local machine, you will require a .env file within the test/e2e/ directory with the following contents:
+
+- `authFile="playwright/.auth/user.json"`
+- `TARGET_HOST=http://localhost:3000`
+- `CAMS_LOGIN_PROVIDER = "mock | okta | none"`
+- `OKTA_USER_NAME=<if provide is okta>`
```
Comment (jamesobrooks) [inline docs/operations/e2e-tests.md:10]:
```suggestion
- `OKTA_USER_NAME=<if provider is okta>`
```

File: docs/operations/e2e-tests.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Running E2E Tests
+
+### Setup for running against locally hosted application
+
+- When running playwright against the locally hosted application, you will wnt to ensure the NodeApi is running against a clean e2e database or the database needed for testing. `COSMOS_DATABASE_NAME=cams-e2e-<envHash>`
+- run the application locally [Running](../running.md)
+- ensure your test/e2e/ .env file has `TARGET_HOST=http://localhost:3000`
+- ensure your test/e2e/ .env file has `CAMS_LOGIN_PROVIDER=mock`
+
```
Comment (jamesobrooks) [inline docs/operations/e2e-tests.md:27]: It might make sense to specify required `.env` settings values for the backend and frontend as well.

**Review by jamesobrooks — CHANGES_REQUESTED** [review]
To make this documentation visible in our Docsify page, please add it to `docs/operations/_sidebar.md`.

**Review by jamesobrooks — APPROVED** [review]

**Review by fmaddenflx — APPROVED** [review]

**Pull request merged**